fixtures/coverall-upstream-compatibility/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "uniffi-fixture-coverall-upstream-compatibility"
+version = "0.22.0"
+authors = ["Firefox Sync Team <sync-team@mozilla.com>"]
+edition = "2021"
+license = "MPL-2.0"
+publish = false
+
+[lib]
+crate-type = ["lib", "cdylib"]
+name = "uniffi_coverall_upstream_compatibility"
+
+[dependencies]
+uniffi = { git = "https://github.com/mozilla/uniffi-rs.git", tag = "v0.23.0" }
+once_cell = "1.12"
+thiserror = "1.0"
+
+[build-dependencies]
+uniffi = { git = "https://github.com/mozilla/uniffi-rs.git", tag = "v0.23.0", features = ["build"] }
+
+[dev-dependencies]
+uniffi = { git = "https://github.com/mozilla/uniffi-rs.git", tag = "v0.23.0", features = ["bindgen-tests"] }
```

Comment:
Isn't this supposed to reference our fork? As I understand `dependencies` and `build-dependencies` are used for scaffolding, and `dev-dependencies` are used to generate foreign language bindings in tests.

Reply:
Let me double check it because I deliberately created tests first before the corrections to be sure that they are actually failing in the predicted way, but I also remember that I had some issues with setting those dependencies (at first I had it just as you suggest) so maybe at some point I've messed it up. I'll double check that with those dependencies the tests are testing what they are supposed to be testing.

Reply:
In the current state, I see that the code is already fixed and only upstream is used to generate the scaffolding. If you want to make sure that the test is actually working, it should be the other way around: `nordsecurity/uniffi-rs` should be used to generate bindings, and there should be no fixes to the code.

Reply:
Thanks for the catch. The tests were indeed not testing anything. Fixed now, and few additional fixes were applied to the code.